### DISCUSSION TITLE
.w2grid change GET method to POST to get records in 2.0

### DISCUSSION THREAD

**Discussion by AlexVDem** [vitmalina/w2ui#2324] [Q&A]
Hi.
I just try to use 2.0 version and to get data from PHP backend I use 
`url: {
  			get: 'records.php',
   			save: 'records.php',
   			remove: 'records.php'
		},`
And, I always get GET request method as payload to receive records in records.php. Even if I use w2utils.settings.dataType = 'JSON'; 
It's strange, because save and remove send POST request to records.php. Is it possible to make it the same - POST for all of them? Thanks.

**Comment by anatoliyrnd** [comment]
And why not use a global array $_REQUEST

**Reply by vitmalina** [reply]
You can intercept the call and set your own method

```js
onRequest(event) {
   event.detail.httpMethod = 'POST'
}
```

for submit
```js
onSubmit(event) {
   event.detail.httpMethod = 'POST'
}
```
